assets/Script/technology/techGame1.js

```diff
@@ -0,0 +1,75 @@
+const { changeScene } = require('../util/sceneUtils');
+const { MENU_SCENE } = require('../constants');
+const MAX_BLOCK_NUMBER = 6;
+
+const generateNumberBlocks = () => {
+    return Math.floor(Math.random() * MAX_BLOCK_NUMBER);
+};
+
+const setupEventHandlers = (that) => {
+  that.homeSprite.on('mousedown', () => {
+    changeScene(MENU_SCENE);
+  });
+};
+
+cc.Class({
+    extends: cc.Component,
+
+    properties: {
+        speed : 200
+    },
+
+    onLoad () {
+        this.homeSprite = this.node.getChildByName('home_button');
+        this.blockSprite = this.node.getChildByName('block')
+
+        var numberBlocks = generateNumberBlocks();
+        var blockPosY = this.blockSprite.getPosition().y;
+        var blockWidth = this.blockSprite.getBoundingBox().size.height;   
```

Comment:
Would this not be `this.blockSprite.getBoundingBox().size.width`?

Reply:
Fixed. Thanks!